submissions/build.gradle.kts

```diff
@@ -15,16 +15,25 @@ extra["springCloudAzureVersion"] = "5.13.0"
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("com.azure.spring:spring-cloud-azure-starter-storage")
+    implementation("com.microsoft.azure:applicationinsights-runtime-attach:3.5.3")
```

Comment:
I think is already in the project conventions?

Reply:
This isn't in project conventions, that's  `applicationinsights-core`. But I reviewed and didn't actually need this one so I removed.